Documentation/building/bootstrapping.md

```diff
@@ -0,0 +1,80 @@
+To build .NET Core by yourself and bootstrap your application with it, follow these steps. 
```

Comment:
What is the doc for? What are the use cases?
How does it overlap with dogfooding / docs on doing CoreCLR+CoreFX change?

Reply:
These docs cover similar topics as well:

https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingYourBuild.md
https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingCoreRun.md

Reply:
Thanks Jan. I agree, both docs cover similar topics but are a) partially not up to date and b) limited by their repository boundaries. My approach tries to provide clear steps what is necessary to build your own custom .NET Core build without bothering with the SDK, NuGet packages and other tools.